src/test/java/com/wikia/webdriver/elements/mercury/pages/login/JoinPage.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.elements.mercury.pages.login;
+
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import lombok.Getter;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.Getter'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)